mono/mini/intrinsics.c

```diff
@@ -119,17 +119,27 @@ llvm_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 			opcode = OP_ABSF;
 		else if (!strcmp (cmethod->name, "Sqrt"))
 			opcode = OP_SQRTF;
+		else if (!strcmp (cmethod->name, "Floor"))
+			opcode = OP_FLOORF;
+		else if (!strcmp (cmethod->name, "Ceiling"))
+			opcode = OP_CEILF;
+		else if (!strcmp (cmethod->name, "FusedMultiplyAdd"))
+			opcode = OP_FMAF;
```

Comment:
I don't think I need to check for SSE41 and FMA capabilities here -- these are high level intrinsics with fallbacks, e.g. I tested it on a Ivy Bridge CPU without FMA -- the following test worked there https://gist.github.com/EgorBo/8f190158748968c9348cc04ee803b2a5